package.json

```diff
@@ -55,6 +55,17 @@
 					"type": "number",
 					"default": 120,
 					"description": "Timeout for installing .NET in seconds."
+				},
+				"dotnetAcquisitionExtension.existingDotnetPath": {
+					"type": {
+						"version": "string",
+						"path": "string"
+					},
+					"default": {
+						"version": "all",
+						"path": "C:\\Program Files\\dotnet"
+					},
```

Comment:
Also, what are your testing plans when you release?  I would hope it includes a quick sanity test of our extension.

Reply:
We have set up manual testing with our vendors, they will be testing with our [sample extension](https://github.com/dotnet/vscode-dotnet-runtime/tree/master/sample), which essentially exposes our commands to the user. I will sanity test locally with the ARM extension as well before releasing.